### ISSUE TITLE
Bug: Accessibility Select Option Groups Multi Selection 

### ISSUE THREAD

**Issue by pavelTre** [sbb-design-systems/sbb-angular#1469]
**Describe the bug**
When you select an option, the status "selected" is displayed. If you navigate away from the field and then back to the selected option, the screen reader does not indicate that the option is selected.

**To Reproduce**
1. Go to ['Stackblitz'](https://stackblitz.com/run?file=src%2Fapp%2Fselect-option-groups-multi-selection-example.ts)
2. Use a Screen Reader example NVDA
3. Open Select Option Groups
4. Select any option
5. Navigate with down or up arrow keys away
6. Navigate back to currently selected option
7. See or hear the error :)

**Expected behavior**
Selected options should be indicated by the screen reader that they are selected

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome, Edge

**Comment by mhaertwig** [comment]
Thanks for reporting this. However, that's not a bug but the intended behaviour of NVDA. From [NVDA Issue #12893](https://github.com/nvaccess/nvda/issues/12893):

> Only announcing one of the states (selected vs not selected) reduces verbosity, the other state is implied.
